CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+cmake_minimum_required(VERSION 3.27)
```

Comment:
That is quite a new version, are any of the new features needed? If not it would be good to use something older that is available on most distros.